bindings/python/setup.py.cmakein

```diff
@@ -16,8 +16,11 @@
 import setuptools
 from setuptools.command.install import install
 
-with open("../../README.md", "r") as fh:
-    long_description = fh.read()
+try:
```

Comment:
this part of commit `Require python3 in CI and have fallback on readme import for setuptools ` could go into the first commit;
it's a fix commit;